docs/installation.rst

```diff
@@ -269,6 +269,25 @@ Also make sure you have musl-gcc in your path::
 Then just build using cargo and the appropriate target::
 
     $ cargo build --target x86_64-unknown-linux-musl
+    
+Build from source (Gentoo, docker)
+==========================
+
+Clone vagga repository
+
+   $ git clone https://github.com/tailhook/vagga && cd vagga
+   
+Describe vagga version
+
+   $ git describe 
+   
+For example: `v0.8.1-19-g372bded`
+
+Compile vagga with dockerized rust:
+
+   $ docker run --rm --user "$(id -u)":"$(id -g)" -e VAGGA_VERSION=v0.8.1-19-g372bded -v "$PWD":/usr/src/myapp -w /usr/src/myapp rust cargo build --release
```

Comment:
If you're already using `$(id -u)` you might want to use `-e VAGGA_VERSION=$(git describe)`, right?

Reply:
in my environment `-e VAGGA_VERSION=$(git describe)` does not work :(

Reply:
I don't know, why it does not work, but without `-e VAGGA_VERSION=v0.8.1-19-g372bded` it drops with complilation error (due VAGGA_VERSION will be empty)